src/engine.rs

```diff
@@ -60,29 +62,33 @@ impl Engine for PbftEngine {
             .expect("This node is not in the peers list, which is necessary")
             as u64;
 
+        let mut pbft_state =
+            DiskStorage::new_with_default("test.json", || PbftState::new(node_id, &config))
```

Comment:
I realize this is WIP, but I still think a name more descriptive than `"test.json"` may be in order here

Reply:
This is what we have done in the past:
https://github.com/hyperledger/sawtooth-raft/blob/master/src/path.rs
https://github.com/hyperledger/sawtooth-raft/blob/master/src/fs_storage.rs#L36

It may also be a good idea to split up the state across several files, so that we are not writing all of state to disk every time we change any part of it.